examples/kafka-bridge/kafka-bridge.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kafka.strimzi.io/v1alpha1
+kind: KafkaBridge
+metadata:
+  name: bridge
+spec:
+  replicas: 1
+  bootstrapServers: my-cluster-kafka-bootstrap:9092
+  image: strimzi/kafka-bridge:latest
```

Comment:
The example should not have the image configuration. It should take it from the CO configuration.